docs/USAGE.md

```diff
@@ -20,7 +20,8 @@
 | `-r`       | `--html_report_name`          | Optional               | Name of the HTML validation report (including `.html` extension).                                                                                                                                                                                            |
 | `-e`       | `--system_errors_report_name` | Optional               | Name of the system errors report (including `.json` extension).                                                                                                                                                                                              |
 | `-n`       | `--export_notices_schema`     | Optional               | Export notice schema as a json file.                                                                                                                                                                                                                         |
-| `-p`       | `--pretty`                    | Optional               | Pretty JSON validation report. If specified, the JSON validation report will be printed using JSON Pretty print. This does not impact data parsing.                                                                                                      |
+| `-p`       | `--pretty`                    | Optional               | Pretty JSON validation report. If specified, the JSON validation report will be printed using JSON Pretty print. This does not impact data parsing.                                                                                                          |
+| `-d`       | `--date`                      | Optional               | Date to simulate when validating, in ISO_LOCAL_DATE format like '2001-01-30'. By default, the current date is used.                                                                                                                                          |
```

Comment:
@jcpitre "Date to simulate when validating" is a bit vague. What about "The date used to validate the feed for time-based rules, e.g [`feed_expiration_30_days`](https://gtfs-validator.mobilitydata.org/rules.html#feed_expiration_date30_days-rule)."

Reply:
Sure. I was just copying the text from @bradyhunsaker's changes, but less vague is always good.